sql/src/test/java/io/crate/integrationtests/PgCatalogITest.java

```diff
@@ -91,4 +91,10 @@ public void testPgDescriptionTableIsEmpty() {
         assertThat(printedTable(response.rows()), is(""));
         assertThat(response.cols(), arrayContaining("classoid", "description", "objoid", "objsubid"));
     }
+
+    @Test
+    public void testPgTypeNameTypeInfoIsAvailable() {
```

Comment:
looks like this is covered already
https://github.com/crate/crate/pull/8512/files#diff-2c56597496c66023d93ded8f9db22150R163

Reply:
removed it. I added this so I can investigate a bug which ended up being irrelevant to the change